backend/src/main/resources/application.yml

```
@@ -10,9 +10,9 @@ server:
 postgres:
   servername: localhost
   port: 5432
-  database: <>
-  username: <>
-  password: <>
+  database: startracker
```

```suggestion
  database: <>
```

I feel like these changes are not needed.